uqcsbot/scripts/wolfram.py

```diff
@@ -0,0 +1,218 @@
+from uqcsbot import bot, Command
+from typing import Iterable, Tuple, Optional
+from base64 import b64decode
+import requests
+import json
+
+APP_ID = b64decode('RzU0S1VBLVVHWTdHR0hWUlg=').decode('utf-8')
```

Comment:
For the jsbot we have used an environment variable for this instead of hardcoding it. Would we be able to load this from an environment variable instead? (with a link on how to get an App ID)